asv_bench/benchmarks/import.py

```diff
@@ -0,0 +1,18 @@
+class Import:
+    """Benchmark importing xarray"""
+
+    def timeraw_import_xarray(self):
+        return """
+        import xarray
+        """
```

Comment:
Nice! For more reading:
https://asv.readthedocs.io/en/v0.5/writing_benchmarks.html#raw-timing-benchmarks